src/scss/sections/hero/hero.scss

```diff
@@ -93,8 +115,20 @@
 .swiper-pagination-custom, .swiper-pagination-fraction {
 	bottom: 0;
 	left: 0;
-	transform: translateY(-410%);
+	transform: translate( 2%,-650% ); //(-410%)
```

Comment:
Ammm.. what are these weird percents doing here? Maybe there is more simple way of positioning?